pkgs/development/libraries/libad9361/default.nix

```diff
@@ -15,11 +15,19 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ libiio ];
 
+  postPatch = ''
+    # Fix iio include path on darwin
```

Comment:
This comment suggests these paths need patch only on darwin, if so, (1) why? (2) Did you report this upstream? or perhaps tried to fix it in their cmake files? (3) I'd also expect this to be an optional attribute only if `stdenv.isDarwin`.

Reply:
See https://github.com/analogdevicesinc/libad9361-iio/blob/43643ab0ef8a3fbdb6c7595f42842f534a291664/test/auto_rate_test_hw.c#L11-L15

Reply:
> See [analogdevicesinc/libad9361-iio@43643ab/test/auto_rate_test_hw.c#L11-L15](https://github.com/analogdevicesinc/libad9361-iio/blob/43643ab0ef8a3fbdb6c7595f42842f534a291664/test/auto_rate_test_hw.c#L11-L15)

Thanks for the link. Perhaps it'd be nice to explain in a comment that here on Nixpkgs we use the same include directory structure as on Linux and that's why it's needed.

Reply:
Done.